--- a/EP/verify.f90
+++ b/EP/verify.f90
@@ -5,2 +4,0 @@
-      use, intrinsic :: ieee_arithmetic, only : ieee_is_nan
-
@@ -67 +65 @@
-         if (ieee_is_nan(sx_err) .or. ieee_is_nan(sy_err)) then
+         if ((sx_err.ne.sx_err) .or. (sy_err.ne.sy_err)) then
@@ -75 +73 @@
-         if (ieee_is_nan(gc_err) .or. gc_err.gt.epsilon) then
+         if ((gc_err.ne.gc_err) .or. gc_err.gt.epsilon) then
--- a/FT/ft.f90
+++ b/FT/ft.f90
@@ -874,2 +873,0 @@
-      use, intrinsic :: ieee_arithmetic, only : ieee_is_nan
-
@@ -1109 +1107 @@
-            if (ieee_is_nan(err) .or. (err .gt. epsilon)) goto 100
+            if ((err.ne.err) .or. (err .gt. epsilon)) goto 100
